examples/nidcpower_source_dc_voltage/NIDCPowerSourceDCVoltage.pinmap

```diff
@@ -2,17 +2,20 @@
 <PinMap xmlns="http://www.ni.com/TestStand/SemiconductorModule/PinMap.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" schemaVersion="1.6">
 	<Instruments>
 		<NIDCPowerInstrument name="DCPower1" numberOfChannels="4">
-			<ChannelGroup name="CommonDCPowerChannelGroup" />
+			<ChannelGroup name="Group1" channels="0,2" />
+			<ChannelGroup name="Group2" channels="1,3" />
 		</NIDCPowerInstrument>
 	</Instruments>
 	<Pins>
 		<DUTPin name="Pin1" />
+		<DUTPin name="Pin2" />
```

Comment:
This is a regression for #224. This example no longer works with single-channel SMUs.